### PR TITLE
create type

### PR DESCRIPTION
**Pull request by NgocVu1994**
create, edit, destroy type

### CODE REVIEW

File: app/controllers/admin/types_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class Admin::TypesController < ApplicationController
+  load_and_authorize_resource
```
Comment (vietmt) [inline app/controllers/admin/types_controller.rb]: except: :show

File: app/models/type.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class Type < ActiveRecord::Base
   has_many :users
+
```
Comment (yongsokheng) [inline app/models/type.rb]: nen validate trong model. presence: true

**Comment by yongsokheng** [comment]
OK

**Comment by hellovietnam93** [comment]
ok

**Pull request closed without merging**